contents/docs/billing/estimating-usage-costs.mdx

```diff
@@ -141,7 +141,8 @@ We don't have data on those yet, unfortunately. In that case, we'd recommend goi
 
 ## How to reduce your PostHog costs
 
-You have two options:
+Each product can be tuned to use only the resources you need. 
 
-- Want to reduce events? Read [How to capture fewer unwanted events](/tutorials/fewer-unwanted-events)
-- Want to reduce recordings? Read [How to only record the sessions you want](/tutorials/limit-session-recordings)
+- For Product analytics, read [how to capture fewer unwanted events](/tutorials/fewer-unwanted-events)
+- For Session replay, read [how to only record the sessions you want](/tutorials/limit-session-recordings)
+- For Feature flags, read [how to make fewer API requests](/docs/feature-flags/common-questions#how-can-i-reduce-the-cost-of-feature-flags)
```

Comment:
I think saying API makes people think it is only available for API, not SDK so just "requests" works. 
```suggestion
- For feature flags, read [how to make fewer flag requests](/docs/feature-flags/common-questions#how-can-i-reduce-the-cost-of-feature-flags)
```